README.rst

```diff
@@ -33,21 +33,57 @@ more VO capabilities into Astropy.
 
 Source code can be found `on GitHub <http://github.com/astropy/pyvo>`_
 
+Installation and Requirements
+-----------------------------
+
+PyVO requires Python 3.8 or later.
+
+The following packages are required for PyVO:
+
+ * `astropy <https://astropy.org>`__ (>=4.0)
+ * `requests <http://docs.python-requests.org/en/latest/>`_
+
+The following packages are optional dependencies and are required for the
+full functionality:
+
+ * mimeparse
+ * pillow
+
+For running the tests, and building the documentation, the following
+infrastructure packages are required:
+
+* `pytest-astropy <https://github.com/astropy/pytest-astropy>`__
+* requests-mock
+* `sphinx-astropy <https://github.com/astropy/sphinx-astropy>`__
+
```

Comment:
This is great but I would start with the easy options first: `pip`. Also, it should mention that it's distributed with `Conda` too. Only after this, I would go with these details.

Reply:
hahh, I landed in the 🐰 🕳️ . I really just wanted to list the necessary info for those building from source, but will add a mention of conda, too.